### ISSUE TITLE
Error after upgrading to Cube v0.35

### ISSUE THREAD

**Issue by UsmanYasin** [cube-js/cube#7951]
**Problem**

I have upgraded my demo environment to v0.35 and getting the following error. Previous version (0.34.62) seem to be working fine. Datasource: MSSQL
```
cube_api-1             | Error: error:0308010C:digital envelope routines::unsupported
cube_api-1             |     at new Hash (node:internal/crypto/hash:69:19)
cube_api-1             |     at Object.createHash (node:crypto:133:10)
cube_api-1             |     at NTLMResponsePayload.ntHash (/cube/node_modules/tedious/lib/ntlm-payload.js:149:19)
cube_api-1             |     at NTLMResponsePayload.ntv2Hash (/cube/node_modules/tedious/lib/ntlm-payload.js:142:23)
cube_api-1             |     at NTLMResponsePayload.lmv2Response (/cube/node_modules/tedious/lib/ntlm-payload.js:130:23)
cube_api-1             |     at NTLMResponsePayload.createResponse (/cube/node_modules/tedious/lib/ntlm-payload.js:72:27)
cube_api-1             |     at new NTLMResponsePayload (/cube/node_modules/tedious/lib/ntlm-payload.js:23:22)
cube_api-1             |     at Connection.message (/cube/node_modules/tedious/lib/connection.js:2183:27)
cube_api-1             |     at Connection.dispatchEvent (/cube/node_modules/tedious/lib/connection.js:1279:15)
cube_api-1             |     at MessageIO.<anonymous> (/cube/node_modules/tedious/lib/connection.js:1139:14)
cube_api-1             |     at MessageIO.emit (node:events:517:28)
cube_api-1             |     at Message.<anonymous> (/cube/node_modules/tedious/lib/message-io.js:46:14)
cube_api-1             |     at Message.emit (node:events:529:35)
cube_api-1             |     at endReadableNT (/cube/node_modules/readable-stream/lib/_stream_readable.js:1094:12)
cube_api-1             |     at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

**Comment by ovr** [comment]
Thank you for submitting the issue!

Fixed & released in [v0.35.1](https://github.com/cube-js/cube/releases/tag/v0.35.1). Could you give it a try?

Thanks

**Comment by UsmanYasin** [comment]
Thanks for the quick turnaround. I have tried the new version, but the error message has now changed. 
I am seeing two different errors now based on the CUBEJS_DB_SSL environment variable.
1. CUBEJS_DB_SSL=false
```
cube_api-1             | Error: Tarn: unsupported option opt.evictionRunIntervalMillis
cube_api-1             |     at QueryQueue.parseResult (/cube/node_modules/@cubejs-backend/query-orchestrator/src/orchestrator/QueryQueue.js:384:13)
cube_api-1             |     at QueryQueue.executeInQueue (/cube/node_modules/@cubejs-backend/query-orchestrator/src/orchestrator/QueryQueue.js:356:19)
cube_api-1             |     at /cube/node_modules/@cubejs-backend/query-orchestrator/src/orchestrator/QueryCache.ts:756:17
cube_api-1             |     at QueryOrchestrator.fetchQuery (/cube/node_modules/@cubejs-backend/query-orchestrator/src/orchestrator/QueryOrchestrator.ts:302:20)
cube_api-1             |     at OrchestratorApi.executeQuery (/cube/node_modules/@cubejs-backend/server-core/src/core/OrchestratorApi.ts:98:20)
cube_api-1             |     at /cube/node_modules/@cubejs-backend/api-gateway/src/gateway.ts:1442:21
cube_api-1             |     at async Promise.all (index 0)
cube_api-1             |     at ApiGateway.getSqlResponseInternal (/cube/node_modules/@cubejs-backend/api-gateway/src/gateway.ts:1440:31)
cube_api-1             |     at /cube/node_modules/@cubejs-backend/api-gateway/src/gateway.ts:1618:28
cube_api-1             |     at async Promise.all (index 0)
cube_api-1             |     at ApiGateway.load (/cube/node_modules/@cubejs-backend/api-gateway/src/gateway.ts:1609:23)
cube_api-1             |     at /cube/node_modules/@cubejs-backend/api-gateway/src/gateway.ts:247:7
```
Seems like this is happening due to the following 3 pool configuration properties in @cubejs-mssql-driver/MSSqlDriver.js . Removing these 3 configurations seem to solve the issue.
```
        evictionRunIntervalMillis: 10000,
        softIdleTimeoutMillis: 30000,
        testOnBorrow: true,
```

2. CUBEJS_DB_SSL=true
```
cube_api-1             | Error: Failed to connect to *****REDACTED*****.com:1433 - unable to verify the first certificate
cube_api-1             |     at QueryQueue.parseResult (/cubejs/packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js:384:13)
cube_api-1             |     at QueryQueue.executeInQueue (/cubejs/packages/cubejs-query-orchestrator/src/orchestrator/QueryQueue.js:356:19)
cube_api-1             |     at /cubejs/packages/cubejs-query-orchestrator/src/orchestrator/QueryCache.ts:756:17
cube_api-1             |     at QueryOrchestrator.fetchQuery (/cubejs/packages/cubejs-query-orchestrator/src/orchestrator/QueryOrchestrator.ts:302:20)
cube_api-1             |     at OrchestratorApi.executeQuery (/cubejs/packages/cubejs-server-core/src/core/OrchestratorApi.ts:98:20)
cube_api-1             |     at /cubejs/packages/cubejs-api-gateway/src/gateway.ts:1442:21
cube_api-1             |     at async Promise.all (index 0)
cube_api-1             |     at ApiGateway.getSqlResponseInternal (/cubejs/packages/cubejs-api-gateway/src/gateway.ts:1440:31)
cube_api-1             |     at /cubejs/packages/cubejs-api-gateway/src/gateway.ts:1618:28
cube_api-1             |     at async Promise.all (index 0)
cube_api-1             |     at ApiGateway.load (/cubejs/packages/cubejs-api-gateway/src/gateway.ts:1609:23)
cube_api-1             |     at /cubejs/packages/cubejs-api-gateway/src/gateway.ts:247:7
cube_api-1             | Orchestrator error: 959684d7-76b1-479e-b72a-0ee91cbbe8a1-span-1 (252ms)
```

**Comment by igorlukanin** [comment]
I believe this issue was fixed in #7977
